set.mm

```diff
@@ -469438,6 +469469,445 @@ Splicing words (substring replacement)
+      dfmgc2lem.4 $e |- ( ph
+             -> A. u e. B A. v e. B ( u .c_ v -> ( G ` u ) .<_ ( G ` v ) ) ) $.
+      dfmgc2lem.5 $e |- ( ( ph /\ x e. A ) -> x .<_ ( G ` ( F ` x ) ) ) $.
+      dfmgc2lem.6 $e |- ( ( ph /\ u e. B ) -> ( F ` ( G ` u ) ) .c_ u ) $.
+      $( Lemma for dfmgc2, backwards direction.  (Contributed by Thierry
+         Arnoux, 26-Apr-2024.) $)
+      dfmgc2lem $p |- ( ph -> F H G ) $=
+        ( vz vw wbr wf wa cv cfv wb wral wcel cproset ad3antrrr simplr ffvelrnd
+        jca adantr ad2antrr simpr ralrimiva wceq fveq2d breq12d rspcdv wi breq1
+        mpd fveq2 breq1d imbi12d breq2 breq2d ffvelrnda eqidd rspc2vd imp prstr
+        syl132anc impbida anasss ralrimivva mgcval mpbir2and ) AHIJUJFGHUKZGFIU
+        KZULUHUMZHUNZUIUMZNUJZWLWNIUNZKUJZUOZUIGUPUHFUPAWJWKUBUCVBAWRUHUIFGAWLF
+        UQZWNGUQZWRAWSULZWTULZWOWQXBWOULZLURUQZWSWMIUNZFUQWPFUQZWLXEKUJZXEWPKUJ
+        ZWQAXDWSWTWOTUSXBWSWOAWSWTUTZVCZXCGFWMIAWKWSWTWOUCUSXCFGWLHAWJWSWTWOUBU
+        SXJVAVAXBXFWOXBGFWNIAWKWSWTUCVDXAWTVEZVAZVCXCBUMZXMHUNZIUNZKUJZBFUPZXGA
+        XQWSWTWOAXPBFUFVFUSXCXPXGBWLFXJXCXMWLVGZULZXMWLXOXEKXCXRVEZXSXNWMIXSXMW
+        LHXTVHVHVIVJVMXBWOXHXBEUMZDUMZNUJZYAIUNZYBIUNZKUJZVKZDGUPEGUPZWOXHVKZAY
+        HWSWTUEVDXBYIWMYBNUJZXEYEKUJZVKYGEDWMWNGGGYAWMVGZYCYJYFYKYAWMYBNVLYLYDX
+        EYEKYAWMIVNVOVPYBWNVGZYJWOYKXHYBWNWMNVQYMYEWPXEKYBWNIVNVRVPXAWMGUQZWTAF
+        GWLHUBVSZVCXBYLULGVTXKWAVMWBFLKWLXEWPOQWCWDXBWQULZMURUQZYNWPHUNZGUQWTWM
+        YRNUJZYRWNNUJZWOAYQWSWTWQUAUSXAYNWTWQYOVDYPFGWPHAWJWSWTWQUBUSXBXFWQXLVC
+        VAXAWTWQUTZXBWQYSXBXMCUMZKUJZXNUUBHUNZNUJZVKZCFUPBFUPZWQYSVKZAUUGWSWTUD
+        VDXBUUHWLUUBKUJZWMUUDNUJZVKUUFBCWLWPFFFXRUUCUUIUUEUUJXMWLUUBKVLXRXNWMUU
+        DNXMWLHVNVOVPUUBWPVGZUUIWQUUJYSUUBWPWLKVQUUKUUDYRWMNUUBWPHVNVRVPXIXBXRU
+        LFVTXLWAVMWBYPYDHUNZYANUJZEGUPZYTAUUNWSWTWQAUUMEGUGVFUSYPUUMYTEWNGUUAYP
+        YAWNVGZULZUULYRYAWNNUUPYDWPHUUPYAWNIYPUUOVEZVHVHUUQVIVJVMGMNWMYRWNPRWCW
+        DWEWFWGAUHUIFGHIJKLMNOPQRSTUAWHWI $.
+    $}
+
+    $d .<_ i u v $.  $d .<_ i x y $.  $d .<_ j $.  $d .<_ m n $.  $d .c_ i j $.
+    $d .c_ j v $.  $d .c_ m u v $.  $d .c_ n $.  $d .c_ x y $.  $d A i x y $.
+    $d A m x y j $.  $d A n y $.  $d B i u v $.  $d B i x y $.  $d B j n $.
+    $d B m $.  $d F i u v $.  $d F i x y $.  $d F j $.  $d F m n $.
+    $d G i u v $.  $d G i x y $.  $d G j $.  $d G m n $.  $d H u v $.
+    $d H x y $.  $d V x y j m n $.  $d W x y j m n $.  $d i ph u v $.
+    $d i ph x y $.  $d m ph $.
+    $( Alternate definition of the monotone Galois connection.  (Contributed by
+       Thierry Arnoux, 26-Apr-2024.) $)
+    dfmgc2 $p |- ( ph -> ( F H G <-> ( ( F : A --> B /\ G : B --> A ) /\ ( ( A.
+      x e. A A. y e. A ( x .<_ y -> ( F ` x ) .c_ ( F ` y ) ) /\ A. u e. B A.
+      v e. B ( u .c_ v -> ( G ` u ) .<_ ( G ` v ) ) ) /\ ( A. u e. B ( F ` ( G
+      ` u ) ) .c_ u /\ A. x e. A x .<_ ( G ` ( F ` x ) ) ) ) ) ) ) $=
```

Comment:
this is difficult to read, it should be formatted a little bit.

Reply:
Yes, I'll split each of the 4 conjuncts on a separated line.